### DISCUSSION TITLE
Using awss3sink in gst template

### DISCUSSION THREAD

**Discussion by ankush-zipteams** [jbg/gst-meet#57] [General]
**Question:**
How can I use the [awss3sink](https://gstreamer.freedesktop.org/documentation/aws/awss3sink.html?gi-language=c) in my recv-pipeline for gstreamer?
When I use it directly in my pipeline description, I end up with this error:

```
2022-11-11T12:01:40.044127Z  WARN on_participant failed: failed to parse recv pipeline participant template

Caused by:
    no element "awss3sink"
```

**Comment by jbg** [comment]
Please, for support questions use Discussions, keep Issues for bug reports.

The error message indicates that you do not have the `awss3sink` GStreamer element. Please check that you have the corresponding package installed. Be aware that you may possibly have more than one GStreamer installation — ensure that the package containing the `awss3sink` element is installed in the same GStreamer installation as you are using with `gst-meet`.
